docarray/typing/tensor/abstract_tensor.py

```diff
@@ -239,6 +239,10 @@ def _docarray_stack(cls: Type[T], seq: Union[List[T], Tuple[T]]) -> T:
         # but mypy doesn't know that, so we ignore it here
         return cls._docarray_from_native(comp_backend.stack(seq))  # type: ignore
 
+    @abc.abstractmethod
+    def __eq__(self, other: object) -> bool:  # type: ignore
```

Comment:
why do you need this one actualy ? I think since it is define at the computational backend level it should work wo

Reply:
I delete those code and just implement it in computational backend which actually works! The test also works fine without it!